traffic_ops/traffic_ops_golang/region/regions_test.go

```diff
@@ -109,3 +109,30 @@ func TestInterfaces(t *testing.T) {
 		t.Errorf("Region must be Identifier")
 	}
 }
+func TestValidation(t *testing.T) {
+	testRegion := tc.Region{
+		DivisionName: "west",
+		ID:           1,
+		Name:         "region1",
+		LastUpdated:  tc.TimeNoMod{Time: time.Now()},
+	}
+	testTORegion := TORegion{Region: testRegion}
+	errs := test.SortErrors(test.SplitErrors(testTORegion.Validate()))
+	expected := []error{}
+
+	if reflect.DeepEqual(expected, errs) {
```

Comment:
You should avoid using `reflect.DeepEqual` to compare error values in tests - and actually I think `golang-lint` warns about this by default. The reason is that errors are consistently appended to outer errors to build an idea of the scope of the error and at what level it occurred. Because of that or just adding more context, error strings change all the time, so using `reflect.DeepEqual` makes tests brittle.

Plus in this particular case, you're only looking for an empty set of errors, so you can easily check `len(errs) != 0` instead.

Reply:
copypasted test, will update it as it seemed weird where i grabbed it from (`divisions_test.go`)